### PR TITLE
XSHIB-159fa8

### PR DESCRIPTION
**Pull request by ShibaversX**

### CODE REVIEW

**Review by ShibaversX — COMMENTED** [review]
[info.json](https://github.com/multiversx/mx-assets/files/14502645/info.json)
logo

**Review by ShibaversX — COMMENTED** [review]
[info.json](https://github.com/multiversx/mx-assets/files/14502878/info.json)
logo

File: tokens/XSHIB-159fa8 [hunk 1]
```diff
@@ -1 +1,11 @@
-
```
Comment (ShibaversX) [inline tokens/XSHIB-159fa8]: logo

Comment (ShibaversX) [inline tokens/XSHIB-159fa8]: logo

**Comment by paulmultiversx** [comment]
Hi, @ShibaversX.

Please use the Assets dApp to properly create a PR: https://assets.multiversx.com/

Once you've created it, don't forget to paste in a comment the signature for the latest commit sha which must be signed with the owner wallet address.

Feel free to use the Sign Message feature from the Utils dApp and paste the signature in a comment for each PR: https://utils.multiversx.com/

Thanks a lot!

**Pull request closed without merging**